NPC Rebakes/npc_features.json

```
@@ -3068,6 +3068,184 @@
       }
     ]
   },
+  {
+    "id": "npc-rebake_npcf_warp_sensors_mirage",
+    "name": "Warp Sensors",
+    "origin": {
+      "type": "Class",
+      "name": "Mirage [K]",
+      "base": true
+    },
+    "locked": false,
+    "type": "Tech",
+    "effect": "The Mirage makes a tech attack against a target within <strong>Sensors</strong>. On a success, they take <strong>{2/3/4} heat</strong> and all attacks they make until the end of their next turn treat their targets as having <strong>soft cover</strong>.",
+    "tags": [
+      {
+        "id": "tg_quick_tech"
+      }
+    ],
+    "tech_type": "Quick",
+    "attack_bonus": [
+      2,
+      4,
+      6
+    ]
+  },
+  {
+    "id": "npc-rebake_npcf_dataveil_mirage",
+    "name": "Dataveil",
+    "origin": {
+      "type": "Class",
+      "name": "Mirage [K]",
+      "base": true
+    },
+    "locked": false,
+    "type": "System",
+    "effect": "By default, the Mirage is <strong>Invisible</strong>. They can transfer this status to another character within line of sight as a quick action. When they do so, the Mirage ceases to be <strong>Invisible</strong>, and as a <strong>quick action</strong> they may transfer it back to themselves or to another character; otherwise, this effect lasts until the Mirage is destroyed.",
+    "tags": []
+  },
+  {
+    "id": "npc-rebake_npcf_glitch_scanners_mirage",
+    "name": "Glitch Scanners",
+    "origin": {
+      "type": "Class",
+      "name": "Mirage [K]",
+      "base": true
+    },
+    "locked": false,
+    "type": "System",
```

```suggestion
    "type": "Tech",
```
This is not consistent with Warp Sensors above

---

It is also missing `tech_type`